stdlib/universe/table_fns_test.go

```diff
@@ -117,11 +119,12 @@ func mustLookup(s values.Scope, valueID string) values.Value {
 func evalOrFail(t *testing.T, script string, mutator flux.ScopeMutator) values.Scope {
 	t.Helper()
 
-	_, s, err := flux.Eval(script, func(s values.Scope) {
-		if mutator != nil {
-			mutator(s)
-		}
-	})
+	_, s, err := flux.Eval(script, context.TODO(), dependencies.NewDefaultDependencies(),
+		func(s values.Scope) {
```

Comment:
must've accidentally took part of Nathaniel's big merge when rebasing here?

Reply:
that was intentional.  All of this rebased off the latest master which includes the recent scope refactor.  

it's more obvious here because I changed the formatting around a bit.